src/styles/main.scss

```diff
@@ -0,0 +1,8 @@
+@import "./utils/variables";
+@import "./blocks/stopwatch";
+@import "./utils/keyframes";
+
+body {
+  display: flex;
+  justify-content: center;
```

Comment:
Maybe also add "align-items: center", because on screens with bigger height your watch is not in center.

https://prnt.sc/abxq0hIyRRlt